content/15.development_procedure.md

```diff
@@ -92,15 +92,26 @@ By queuing up pull requests for merging, it allows maintainers to mark a set of
 ### Code Styling and Linting
 
 For code included in ``yt``, a set of styles are enforced.
-We use the automated code styling tool [``black``](https://black.readthedocs.io/en/stable/), which has been designed to maximize readability with as few free parameters as possible.
-(In many ways, the fact that most of the ``yt`` code developers did not utilize this style before it was enforced likely enabled its uptake, as it was seen as an choice that "made everyone compromise.")
-In addition to code formatting, other automated methods for code consistency are applied; at present, this includes ensuring that all ``import`` statements are sorted (according to alphabetical order within a first/second/third-party categorization).
-Past iterations of this automated formatting have also included ensuring that ["f-strings"](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals) are used wherever possible, but at present we have slightly loosened this requirement.
 
-For the purposes of checking for common code mistakes, we apply [``flake8``](https://flake8.pycqa.org/en/latest/), which reports unused variables, undefined variables, and other style violations that cannot be automatically corrected.
+We rely on the [``pre-commit``](https://pre-commit.com) framework, which enables automated checks as well as automatic fixes at commit time.
+This tool is an opt-in so not every "drive-by" contributor has to learn and install it, but continuous linting is provided by [pre-commit.ci](https://pre-commit.ci) so styling errors cannot slip in.
 
-All changes that can be applied via automation (specifically, code formatting) are accessible from within the GitHub pull request interface.
-This allows "drive-by" contributions to have their pull requests updated inline by an automated process, reducing the need to manually install packages to apply the changes.
+We configure `pre-commit` to run a mixed bag of formatters and static checkers.
```

Comment:
```suggestion
We configure `pre-commit` to run a mixture of formatters and static checkers.
```

("mixed bag" is often used slightly differently, to communicate a value judgment)

Reply:
I thought it did, but wasn't sure, thanks for having my back !